dataland-framework-toolbox/scripts/convert-excel-to-csv.sh

```diff
@@ -19,7 +19,9 @@ set -euxo pipefail
 # false - Remove spaces --> No
 # -1 - Export all sheets
 
-libreoffice --headless \
+"/c/Program Files/LibreOffice/program"/scalc.exe --headless \
   --convert-to csv:"Text - txt - csv (StarCalc)":44,34,UTF8,1,,1033,false,true,false,false,false,-1 \
   --outdir "$2" \
-  "$1"
\ No newline at end of file
+  "$1"
+
+  # TODO revert to linux command before merging to main?
```

Comment:
Resolve this TODO

Reply:
Well, shall we revert it to the linux command or not?
Or shall we do a check on the OS and run the correct command accordingly?

Reply:
I dont know how much effort it is, to determine the OS and use the corresponding commands. If it is not to much effort i would like it to determine the OS